components/CaseStatus/CaseStatusView.tsx

```diff
@@ -29,7 +29,7 @@ const CaseStatusView = ({ person }: Props): React.ReactElement => {
               key={status.id}
             >
               {getTypeString(status.type as keyof typeof valueMapping)} -{' '}
-              {status.subType}
+              {status.fields.map((val) => val.selectedOption.name)}
```

Comment:
I might not understand correctly the context but this will return an array of strings inside JSX.

If you want to print them (comma separated) something like `{status.fields.map((val) => val.selectedOption.name).join(',')}` might be what you want?

Maybe what you have currently is fine and I'm just lacking context.

Reply:
So, does not really matter for now as we got only one result.

I've added your suggestion anyways as it's a good one, and will help when we're gonna load multiple values